src/main/kotlin/no/skatteetaten/aurora/boober/utils/DeploymentConfigUtils.kt

```diff
@@ -5,5 +5,14 @@ import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
 import com.fasterxml.jackson.module.kotlin.readValue
 import io.fabric8.openshift.api.model.DeploymentConfig
 
+fun DeploymentConfig.findImageChangeTriggerTagName(): String? {
```

Comment:
Hmm... strengt tatt bør man kanskje validere at ?.first() i dette tilfellet (navnet på imagestreamen) er det samme som ImageStreamen man forsøker rulle ut? Koden i dag benytter seg av en implisitt sammenheng mellom dc.name og is.name som kanskje ikke alltid vil gjelde? Mulig det holder å notere seg dette bak øret og at man ikke gjør noe med det nå.

Reply:
Ja, enig. Denne må nok valideres.

Reply:
Egentlig så burde vi bruke labels når vi matcher objekter og ikke name direkte. Det er jeg helt enig i.